components/DaysOfTheWeek/DayOfTheWeekButton.jsx

```diff
@@ -14,6 +14,15 @@ export default function DaysOfTheWeekButton ({ label, onClick, isActive }) {
       colorScheme="blue"
       variant={isActive ? "solid" : "outline"}
       onClick={(e) => onClick(e.target.innerText)}
+      _disabled={{
```

Comment:
Hi @OlaShabalina :wave: 

Frontend newbie question: does the `underscore` in `_disabled` object have a significance? 
Is it a special variable/object in Chakra? 

Thx in advance for your teachings :bow:

Reply:
It's an interesting question which I didn't think of myself, I got it from another button existing in the project. For the sake of experiement I just tested what happens if I remove it - it works but doesn't apply styling anymore. If you find out how to apply styling to it - we can remove `underscore`. 😄
![Screenshot 2022-03-23 145515](https://user-images.githubusercontent.com/88268603/159620913-d6f1abb8-571a-4a88-9867-6cc2e7563c19.png)

Reply:
Hi @OlaShabalina :wave: 

I think I've found something about `_disabled` in the Chakra docs:
- `_disabled` is a [Pseudo Style Prop](https://chakra-ui.com/docs/styled-system/features/style-props#pseudo)
- it can be used with `<Button>` and other components to style a component in a particular way when disabled. While the docs didn't specifically give an example of using `_disabled`, they did use other styled props in examples [here](https://chakra-ui.com/docs/components/form/button)

mystery solved, I think!